File: cpp/25d_1703A_YesOrYes.cpp
#include<bits/stdc++.h>

int main(){
    int t;
    std::string temp;
    std::vector<std::string> ans;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>temp;
        for (int i = 0; i < 3; i++){
            temp[i] = tolower(temp[i]);
        }
        if(temp.compare("yes")){
            ans.push_back("NO");
        }
        else{
            ans.push_back("YES");
        }
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/25d_1703A_YesOrYes.py
def main():
    t = int(input())
    ans = []
    for _ in range(t):
        temp = input().strip().lower()
        if temp == "yes":
            ans.append("YES")
        else:
            ans.append("NO")
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()